in_out.py

```diff
@@ -651,14 +672,24 @@ def get_cls(par,par_vary,sign_vary) :
         dict_l=read_cls(par,clf_lensed)
     else :
         dict_l=dict_t
-    cl_tt=dict_t['cl_tt']; cl_ee=dict_t['cl_ee']; cl_te=dict_t['cl_te']; cl_bb=dict_t['cl_bb'];
-#    cl_tt=dict_l['cl_tt']; cl_ee=dict_l['cl_ee']; cl_te=dict_l['cl_te']; cl_bb=dict_l['cl_bb'];
+    # cl_tt=dict_t['cl_tt']; cl_ee=dict_t['cl_ee']; cl_te=dict_t['cl_te']; cl_bb=dict_t['cl_bb'];
+    cl_tt=dict_l['cl_tt']; cl_ee=dict_l['cl_ee']; cl_te=dict_l['cl_te']; cl_bb=dict_l['cl_bb'];
     if par.has_cmb_p :
         cl_bb*=0
     cl_pp=dict_t['cl_pp']; cl_tp=dict_t['cl_tp']; cl_ep=dict_t['cl_ep'];
     cl_td=dict_t['cl_td']; cl_tl=dict_t['cl_tl']; cl_pd=dict_t['cl_pd']; cl_pl=dict_t['cl_pl'];
     cl_dd=dict_t['cl_dd']; cl_dl=dict_t['cl_dl']; cl_ll=dict_t['cl_ll'];
 
+    # Apply multiplicative bias shift
+
+    # cl_tl[:,bin_idx] *= (1+m)
+    cl_dl[:,:,bin_idx] *= (1+m)
+    cl_pl[:,bin_idx] *= (1+m)
+    cl_ll[:,:,bin_idx] *= (1+m)**2
+    cl_ll[:,bin_idx,:] *= (1+m)**2
+    cl_ll[:,bin_idx,bin_idx] /= (1+m)**2
+
```

Comment:
I don't understand the **2 here. In principle each lensing bin receives one power of (1+m), right?